dev-tools/mage/gotool/go.go

```diff
@@ -0,0 +1,255 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This file is verbatim copied from https://github.com/elastic/go-txfile/tree/master/dev-tools/lib/mage/gotool